_rules/6cbcbe.md

```diff
@@ -0,0 +1,271 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names serve an equivalent purpose.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characaters) [accessible names](#accessible-name) that does not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name](#accessible-name).
+
+### Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource](#same-resource) or to different resources that fulfill an [equivalent purpose](#equivalent-purpose).
+
+**Note:** Resolving the links includes potential redirects.
```

Comment:
What's should happen if redirects are delayed? Consider:

```
<a href="/foo/a.html">Foo</a>
<a href="/foo/b.html">Foo</a>
```

If `/foo/b.html` redirects to `/foo/a.html` after 5 seconds, or perhaps after 20 hours, does the expectation hold?

Reply:
Delayed redirects should, as I see it, be a failure under a different success criterion (see https://auto-wcag.github.io/auto-wcag/rules/SC2-2-1+SC2-2-4-meta-refresh.html). 

What if we limit it to redirects that happen instantly?

Reply:
Sounds good to me!

Reply:
Now redirects have to be instant, and the redirect note is no longer a note, but part of the expectation. 
Failed testcase added, and Passed redirect testcase now specifies that the redirect is instant.